.github/workflows/build-test-release.yml

```diff
@@ -14,6 +14,9 @@ jobs:
     steps:
       - name: Checkout Repo
         uses: actions/checkout@v3
+        with:
+          # check out full history
+          fetch-depth: 0
```

Comment:
This is needed for the `changeset status` to be able to work, otherwise it won't find the reference to `origin/main`